cache/lru_cache.cc

```diff
@@ -741,7 +787,7 @@ void LRUCache::WaitAll(std::vector<Handle*>& handles) {
       }
       sec_handles.emplace_back(lru_handle->sec_handle);
     }
-    secondary_cache_->WaitAll(sec_handles);
+    options_.secondary_cache->WaitAll(sec_handles);
```

Comment:
I think there are more places that we use "secondary_cache_->" for lookup and insertion.

Reply:
Those are done within the shard, which has a secondary_cache_ element.